decentraland/unity-renderer
# feat: limit green blockers at distance 2 for worlds

## What does this PR change?

Solves #3553 

The current Worlds implementation allows bigger scene areas than 2x2. But to get this started, a limit was implemented for any given scene at a distance of x Parcels, currently set at 2. 

Once the limit is applied to worlds, this will keep working.

Also, took the opportunity to due name refactoring (RealmsPlugin to RealmPlugin) as requested in the [Worlds ADR](https://fdc23f87.adr-cvq.pages.dev/adr/ADR-101)

## How to test the changes?

1. Enter `/changerealm menduz.dcl.eth` in the chat
2. You are now in a World; a reef. Green blockers should be 2 parcels away. No empty parcel should be loaded (meaning, no parcel with trees or rocks). All should be empty except for the reef
4. Go back to `/changerealm loki`. Walk to load any parcel. Green blockers should work normally
5. Enter Decentralnd using the link https://play.decentraland.zone/?renderer-branch=feat/limit-green-blockers-at-distance-2&realm=menduz.dcl.eth. Green blockers should be 2 parcels away. 

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
